shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -45,7 +49,17 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
   }
 
   _openFilePicker = () => {
-    showImagePicker({mediaType: isIOS ? 'mixed' : 'photo'}, response => {
+    if (isIOS) {
+      this._showNativeImagePicker('mixed')
+    } else {
+      // On Android "mixed" mode doesn't work with our library, so we have to ask the user
+      // if they want photo or video
+      this._toggleShowingMenu('filepickerpopup')
+    }
+  }
+
+  _showNativeImagePicker = (mediaType: string) => {
+    showImagePicker({mediaType}, response => {
```

Comment:
Can you change the `Title`s in this options object in the video case?
<img width="337" alt="image" src="https://user-images.githubusercontent.com/11968340/43649475-4411f5d2-970b-11e8-837c-0a1e1ac85b9e.png">